BugBountyFE/src/app/components/bug-report-form/bug-report-form.component.ts

```diff
@@ -29,10 +30,16 @@ export class BugReportFormComponent implements OnInit {
     severity: new FormControl('', Validators.required),
   });
 
-  constructor(http:HttpClient, private bugReportService : BugReportService) { }
+  constructor(
+      private bugReportService : BugReportService,
+      private router:Router
+      ) { }
 
   ngOnInit(): void {
-       
```

Comment:
You have a merge conflict concerning this code. Please fix it as I'm not sure what you want to keep.

Reply:
Ok I resolved the merge conflict